apps/src/music/player/types.ts

```diff
@@ -4,6 +4,8 @@ import {Effects} from './interfaces/Effects';
 export interface SampleEvent {
   // 1-based playback position in measures
   playbackPosition: number;
+  // How early to play the sample, in which case it has extra sound that should play before its scheduled time.
+  early?: number;
```

Comment:
Maybe `pickup`? Also could we specify the unit (assuming it's measures like `playbackPosition`)?

Reply:
Interseting.  Have updated to `pickupLength`.  Decided against `anacrusis` too!